src/debug.c

```diff
@@ -776,6 +791,11 @@ NULL
     {
         server.active_expire_enabled = atoi(c->argv[2]->ptr);
         addReply(c,shared.ok);
+    } else if (!strcasecmp(c->argv[1]->ptr,"packed_threshold") &&
+    c->argc == 3)
+    {
+        packed_threshold = atoi(c->argv[2]->ptr);
+        addReply(c,shared.ok); //fridapacked_threshold
```

Comment:
not sure what the comment was for, probably a leftover?
```suggestion
        addReply(c,shared.ok);
```